daphne_worker/src/config.rs

```diff
@@ -276,10 +270,23 @@ impl<D> DaphneWorkerConfig<D> {
             .durable_object(binding)
     }
 
-    pub(crate) fn get_hpke_secret_key_for(&self, hpke_config_id: u8) -> Option<&HpkeSecretKey> {
-        for hpke_secret_key in self.hpke_secret_key_list.iter() {
-            if hpke_config_id == hpke_secret_key.id {
-                return Some(hpke_secret_key);
+    #[allow(dead_code)]
```

Comment:
For this, I've tacked a good amount of time after changing `#[allow(dead_code)]` to `#[cfg(test)]`, as this is the only function that uses `HpkeConfig`.
Rust complains every time I try to include `HpkeConfig` only in the scope of this function, but I cannot seem to do it.
At the end, I've removed the test code entirely, since it is not even called by any function.
Let me know if I should revert this.